dist/images/start-db.sh

```diff
@@ -74,6 +74,12 @@ function is_clustered {
   return 1
 }
 
+#### first  try to boot ovn-leader-checker
+#### we put kube-ovn-leader-checker at the same place of ovn-cental
+#### seperate pod of is also ok
+
+/kube-ovn/kube-ovn-leader-checker &
```

Comment:
it should replace the last tail command and always run

Reply:
i modified codes just like you said.